CodeWritingCow/cnse
# API Assignment 02: Containerize Voter API and add Redis cache

## Description

This PR containerizes the voter API from Assignment 1. It also replaces in-memory Go storage with Redis caching.

## How Has This Been Tested?

1. Go to voter-container/voter-api folder in a terminal window
2. Run build-docker.sh to containerize Voter API
3. Run the command `docker compose up` to start the API
4. Run the test file voter-container/voter-api/tests/voter_api_test.go
5. Verify that all tests pass
